testsuite/shared/src/main/java/org/jboss/as/test/integration/management/extension/blocker/BlockerExtension.java

```diff
@@ -212,6 +209,16 @@ public void handleResult(OperationContext.ResultAction resultAction, OperationCo
                         }, OperationContext.Stage.RUNTIME);
                         break;
                     }
+                    case SERVICE_STOP: {
```

Comment:
Please add a big comment here explaining that this usage is not part of the main purpose of BlockerExtension and instead is to satisfy a different PreparedResponseTestCase. Otherwise it will be quite confusing later on if we ever try and get the original intent of SERVICE_STOP to work.